### PR TITLE
Export pressScale and focusRing utilities from S2

### PR DESCRIPTION
**Pull request by devongovett**
A lot of import changes but this is just exporting `pressScale` (from `@react-spectrum/s2` since it is used at runtime) and `focusRing` from (`@react-spectrum/s2/style` since it is a macro). We could export additional style macro utils from here in the future.

### CODE REVIEW

File: packages/@react-spectrum/s2/src/style-utils.ts [hunk 1]
```diff
@@ -13,16 +13,6 @@
 import {CSSProperties} from 'react';
 import {StyleString} from '../style/types';
```
Comment (ktabors) [inline packages/@react-spectrum/s2/src/style-utils.ts:15]: Should we also expose` getAllowedOverrides()` and `styleProps` from this file?

**Review by ktabors — APPROVED** [review]

**Review by snowystinger — APPROVED** [review]

File: packages/@react-spectrum/s2/src/style-utils.ts (hunk 1 above)
Comment (devongovett) [reply packages/@react-spectrum/s2/src/style-utils.ts:15]: I think we'll keep those internal for now. It's pretty easy to pass in an array to the `style` macro, and different components may have different styles that they want to allow. I exported the StyleString type here so you can use that for now.

**Comment by ktabors** [comment]
What about centerbaseline()? If logic button had an icon it would need that.

**Review by reidbarber — DISMISSED** [review]

**Comment by dannify** [comment]
> What about centerbaseline()? If logic button had an icon it would need that.

No icons in logic button that we know of.

**Comment by devongovett** [comment]
hmm I'm not sure LogicButton supports icons. We can consider exporting `<CenterBaseline>` in the future potentially.

**Review by dannify — DISMISSED** [review]

**Comment by ktabors** [comment]
> hmm I'm not sure LogicButton supports icons. We can consider exporting `<CenterBaseline>` in the future potentially.

Sorry for the confusion, I meant to say it doesn't support icons, but another component might have both icon and text and we'd need `<CenterBaseline />` for it.

**Review by snowystinger — APPROVED** [review]

**Review by ktabors — APPROVED** [review]

**Pull request merged**